frontend/src/components/translation/SendForReviewModal.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import {
+  Box,
+  Button,
+  Flex,
+  Heading,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalCloseButton,
+  Text,
+} from "@chakra-ui/react";
+
+type SendForReviewModalProps = {
+  sendForReview: boolean;
+  onSendForReviewClick: () => void;
```

Comment:
nit: this function name should mention something about closing, id be happy with just `onClose` or `closeSendForReviewModal` or something silmilar